### ISSUE TITLE
Feature Request: Restart Recording button

### ISSUE THREAD

**Issue by HeCorr** [ShareX/ShareX#7255]
Often when recording something with ShareX, I accidentally mess up what I'm recording and have to start over, sometimes more than once, so it would be really helpful to have a "Restart" button that aborts the current recording and resets the countdown timer, keeping the same selected region.

Instead of adding a fourth button (since there are already three large ones plus a timer), you could instead turn the `Abort` button into the `Restart` button during the recording, which when clicked makes the Abort button appear again (during the countdown) so the user would press twice at the same place in order to abort the recording.

![sharex_restart_button](https://github.com/ShareX/ShareX/assets/75134774/3f2959fd-296b-423b-8e28-3316a107923a)

**Comment by sebastienbo** [comment]
Often these buttons are also not reachable, for example a zoom call hides thhese buttons which makes it impossible o control the recording. And the red recording button in the systreay does not have this stop/pause/restart options. And i yo uaccidently clicked the red button you stoped your recording without a warning ... It should ask a confirmation to stop the recording or show a menu with the stop/pause/restart options

**Comment by shodanx2** [comment]
I would like both abort and restart

But yeah, restart would be great when you need to time the beginning of your capture. 
So it would be great to setup the capture, then make the thing happen and press "reset"
So you don't have that "millenial pause" at the beginning of the screen recording while setting stuff up.
I also use losslesscut to trim that, but that's an extra step that could be avoided here. Also losslesscut only cuts on keyframes so there is often a little bit extra, and that's a big deal when you need to stay under that 25MB limit !

**Comment by DevilXD** [comment]
+1 from me on the restart feature, I just came here to suggest the exact same thing, but found this issue instead.

**Comment by shodanx2** [comment]
I would also a  a global keyboard shortcut for these extra function which would only be enabled during a capture

For me video capture is 
shift+printscreen (to start and then end capture)
shift+scrolllock for pausing
shift+break for abort
shift+home for restart
shift+insert for "insert cutpoint" (*.llc file format in same folder as target *.mp4 file see losslesscut format)

**Comment by shodanx2** [comment]
> Often these buttons are also not reachable, for example a zoom call hides thhese...

Yes ! I just realize, there should be a place on this little window you can click and drag to move it around.
And if it's below the bottom of the screen, it should move inside the capture area if it has to, it's ok because we'd have a restart button, you can drag it away from the capture area and hit restart, at least now they don't become inaccessible.

I would also like a check box on this window, to "record immediately" or not. Not a setting buried in the settings. Sometimes I have to wait for the thing I want to record, so I want to just setup the region, and start at the right time, so I don't have to crop the video, which often doesn't work because of keyframes !

This issue should be marked as STALE not NOT PLANNED, turn automatic "NOT PLANNED" thing... 
Because I didn't get a message this was becoming "STALE" last week.
